src/Features/Core/Portable/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -13,8 +13,8 @@
 
 namespace Microsoft.CodeAnalysis.RemoveUnnecessaryImports
 {
-    internal abstract class AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer :
-        DiagnosticAnalyzer, IBuiltInAnalyzer
+    internal abstract class AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer
+        : DiagnosticAnalyzer, IBuiltInAnalyzer, IInProcessAnalyzer
```

Comment:
why is this inproc?

Reply:
I don't know. We have a hypothesis that .editorconfig didn't work out of proc when it was implemented.

Reply:
Hazarding a guess; I wonder if this was expensive to run everywhere or it was too noisy, so we restricted that to be "open file" and then that became a bad proxy for being inproc.